src/screens/AirdropDashboard/AirdropDashboardScreen.tsx

```diff
@@ -46,9 +49,14 @@ export const AirdropDashboardScreen: FC<Props> = ({
   error,
   route,
 }) => {
+  useEffect(() => {
+    getUsersQuantity();
```

Comment:
this fires OK, but saga that should watch for this action getting dispatched, is not getting triggered. Help appreciated.